packages/paste-core/layout/grid/src/Column.tsx

```diff
@@ -2,29 +2,32 @@ import * as React from 'react';
 import * as PropTypes from 'prop-types';
 import {styled, compose, flexbox, layout, space} from '@twilio-paste/styling-library';
 import {ResponsiveProp} from '@twilio-paste/style-props';
-import {ColumnProps, ColumnStyleProps} from './types';
-import {getStackedColumns, getColumnOffset, getColumnSpan} from './utils';
-
-export const getColumnStyles = (props: ColumnProps): ColumnStyleProps => {
+import type {ColumnProps, ColumnStyleProps} from './types';
```

Comment:
To clarify, this file isn't used in the actual Grid right? Is it for testing the utils only?

Reply:
Do you mean the `Column.tsx` file? No this is used to create the grid columns.